luma/oled/device.py

```diff
@@ -125,7 +125,8 @@ def __init__(self, serial_interface=None, width=128, height=64, rotate=0, **kwar
         settings = {
             (128, 64): dict(multiplex=0x3F, displayclockdiv=0x80, compins=0x12),
             (128, 32): dict(multiplex=0x1F, displayclockdiv=0x80, compins=0x02),
-            (96, 16): dict(multiplex=0x0F, displayclockdiv=0x60, compins=0x02)
+            (96, 16): dict(multiplex=0x0F, displayclockdiv=0x60, compins=0x02),
+            (64, 48): dict(multiplex=0x2F, displayclockdiv=0x80, compins=0x12)
         }.get((width, height))
```

Comment:
can we someday place these magic constants in some sort of class or more descriptive object?